src/Conf.ml

```diff
@@ -134,6 +134,19 @@ let escape_strings =
           default
       & info ["escape-strings"] ~doc ~env)
 
+let ifthenelse =
+  let doc =
+    "Format for if-then-else. Can be set in a config file with an \
+     `if-then-else {kw-first,compact}` line."
```

Comment:
I'm not sure it is worth it to abbreviate keyword to kw, but don't feel strongly either way.